src/main/java/eod/warObject/character/concrete/red/AssaultTeamLeader.java

```diff
@@ -54,34 +56,61 @@ public String getName() {
         return "特攻隊隊長";
     }
 
-    @Override
-    public void onEventOccurred(GameObject sender, Event event) {
-        super.onEventOccurred(sender, event);
-        if(event instanceof ObjectEnterEvent) {
-            if(((ObjectEnterEvent) event).getObject() == this) {
-                addAttack(1);
-                addHealth(1);
-            }
+    private class OwnedAbilities implements EventReceiver {
+        private AssaultTeamLeader holder;
+        private ArrayList<Class<? extends Event>> canHandle;
+
+        public OwnedAbilities(AssaultTeamLeader holder) {
```

Comment:
Same as [this](https://github.com/Elegy-Composer/EOD-DM/pull/35/files#r430117491)

Reply:
fixed